ManageWiki.php

```diff
@@ -286,72 +274,72 @@
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:CharInsert',
 			'var' => 'wmgUseCharInsert',
 			'conflicts' => false,
-			'requires' => false,
-			'restricted' => false,
+			'requires' => [],
 		],
 		'cite' => [
 			'name' => 'Cite',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:Cite',
 			'var' => 'wmgUseCite',
 			'conflicts' => false,
-			'requires' => false,
-			'restricted' => false,
+			'requires' => [],
 		],
 		'citethispage' => [
 			'name' => 'CiteThisPage',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:CiteThisPage',
 			'var' => 'wmgUseCiteThisPage',
 			'conflicts' => false,
-			'requires' => false,
-			'restricted' => false,
+			'requires' => [],
 		],
 		'citoid' => [
 			'name' => 'Citoid',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:Citoid',
 			'var' => 'wmgUseCitoid',
 			'conflicts' => false,
-			'requires' => 'cite',
-			'restricted' => false,
+			'requires' => [
+				'extensions' => [
+					'cite',
+					'visualeditor',
+				],
+			],
 		],
 		'codeeditor' => [
 			'name' => 'CodeEditor',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:CodeEditor',
 			'var' => 'wmgUseCodeEditor',
 			'conflicts' => false,
-			'requires' => false,
-			'restricted' => false,
+			'requires' => [],
 		],
 		'codemirror' => [
 			'name' => 'CodeMirror',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:CodeMirror',
 			'var' => 'wmgUseCodeMirror',
 			'conflicts' => false,
-			'requires' => false,
-			'restricted' => false,
+			'requires' => [],
 		],
 		'collapsiblevector' => [
 			'name' => 'Collapsible Vector',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:CollapsibleVector',
 			'var' => 'wmgUseCollapsibleVector',
 			'conflicts' => false,
-			'requires' => false,
-			'restricted' => false,
+			'requires' => [],
 		],
 		'collection' => [
 			'name' => 'Collection (PDF)',
 			'linkPage' => 'https://mediawiki.org/wiki/Extension:Collection',
 			'var' => 'wmgUseCollection',
 			'conflicts' => false,
-			'requires' => 'electronpdfservice',
-			'restricted' => true,
```

Comment:
was this just restricted for the ext requirement?

Reply:
Ah yes, i think so. This should now work (so no longer needs restricting).